commit 4ffdbcbd4e062ca9a822d49432ce2618d0006a8a
Merge: 443b648 5c8107b
Author: Bernhard Kubicek <kubicek@gmx.at>
Date:   Sun Nov 9 20:38:30 2014 +0100

    Merge pull request #1138 from filipmu/Marlin_v1
    
    Comment out the FILAMENT_SENSOR define by default in config.

diff --git a/Marlin/Configuration.h b/Marlin/Configuration.h
index d2aba19..2702024 100644
--- a/Marlin/Configuration.h
+++ b/Marlin/Configuration.h
@@ -777,7 +777,9 @@ const bool Z_MAX_ENDSTOP_INVERTING = true; // set to true to invert the logic of
  * 301 - Rambo  - uses Analog input 3
  * Note may require analog pins to be defined for different motherboards
  **********************************************************************/
-#define FILAMENT_SENSOR
+// Uncomment below to enable
+//#define FILAMENT_SENSOR
+
 #define FILAMENT_SENSOR_EXTRUDER_NUM	0  //The number of the extruder that has the filament sensor (0,1,2)
 #define MEASUREMENT_DELAY_CM			14  //measurement delay in cm.  This is the distance from filament sensor to middle of barrel
 
